packages/storybook/stories/va-file-input-uswds.stories.tsx

```diff
@@ -170,7 +174,7 @@ const CustomValidationTemplate = ({
 
       const reader = new FileReader();
       reader.onloadend = () => {
-        const contents = reader.result;
+        const contents = reader.result as String;
```

Comment:
I think since we're always reading the result here on load end, it'll likely always be a string. So instead of casting it to a string we could probably do something like `const contents: String` instead. If we're not 100% confident the result will always be a string, then we could do something like `const contents: String = result.readAsText(file)`

Reply:
I wasn't able to make `contents: String` work here, nor was I able to make `.readAsText` work, so I'm just going to leave it with `as String` for now.